central/scannerdefinitions/handler/handler_test.go

```diff
@@ -200,11 +201,11 @@ func (s *handlerTestSuite) TestServeHTTP_Online_ZSTD_Bundle_Get() {
 	assert.Equal(t, http.StatusInternalServerError, w.Code)
 
 	// Should get mapping json file from online update.
-	req = s.getRequestWithJSONFile(t, "dev")
+	req = s.getRequestWithVersionedFile(t, "dev")
 	w.Data.Reset()
 	h.ServeHTTP(w, req)
 	assert.Equal(t, http.StatusOK, w.Code)
-	assert.Equal(t, "application/zstd", w.Header().Get("Content-Type"))
+	assert.Equal(t, "application/octet-stream", w.Header().Get("Content-Type"))
```

Comment:
I tried "application/zstd" but not working. Any suggestions?

Reply:
`http.ServeContent` will attempt to set `Content-Type` based on the file extension. If it can't figure it out, then it reads the first few bytes and sets the `Content-Type` accordingly.

`application/zstd` does not seem to be a standard type, so it read the first few bytes and determined the file is some binary encoded file

```
// If the response's Content-Type header is not set, ServeContent
// first tries to deduce the type from name's file extension and,
// if that fails, falls back to reading the first block of the content
// and passing it to DetectContentType.
// The name is otherwise unused; in particular it can be empty and is
// never sent in the response.
```

https://pkg.go.dev/net/http#ServeContent

To resolve this, you should explicitly set `Content-Type: application/zstd` in the header